code/game/objects/items/plushes.dm

```diff
@@ -676,3 +676,23 @@
 	item_state = "moffplush"
 	attack_verb = list("fluttered", "flapped")
 	squeak_override = list('sound/voice/moth/scream_moth.ogg'=1)
+	var/souls = 0
+
+/obj/item/toy/plush/moth/suicide_act(mob/living/user)
+	user.visible_message("<span class='suicide'>[user] begins staring deeply into the eyes of [src]!  It looks like [user.p_theyre()] trying to commit suicide!</span>")
+	if(do_after(user, 50, target = user) && !user.hellbound)
```

Comment:
use "user.mind.hasSoul" instead of "user.hellbound", because liches are a thing (and they don't become hellbound when they stuff their soul into a phylactery)

Reply:
also, if they fail specifically because they don't have a soul, not because they were interrupted, the message should say that the plushie tried to consume their soul, but couldn't find one (or something like that), not that they just "couldn't do it"

Reply:
In light of Cobby wanting to keep shame results minimal I'm removing it entirely. It doesn't really feel like it adds much besides keeping to the theme of the 'soul', and these are just outlier instances anyway.
They can die like anyone else, and I'll just try to tweak the wording so it doesn't step on that theme.